src/main/java/com/google/sps/servlets/AddSchoolInfo.java

```diff
@@ -0,0 +1,159 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.FetchOptions;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.Filter;
+import com.google.appengine.api.datastore.Query.FilterOperator;
+import com.google.appengine.api.datastore.Query.FilterPredicate;
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/add-info")
+public class AddSchoolInfo extends HttpServlet {
+
+  private Key existingSchoolKey;
+  private Key existingCourseKey;
+  private Key existingProfessorKey;
+
+  private Key newSchoolKey;
+  private Key newCourseKey;
+  private Key newProfessorKey;
+  private Key newTermKey;
```

Comment:
Do these actually need to live for the duration of this Servlet? 
Typically I'd advise keeping parameters close to where they're used and as locally scoped as possible.

Reply:
I was able to get rid of the new...Key variables but I couldn't figure out how to get rid of the existing key variables